### ISSUE TITLE
Remove Policy Package

### ISSUE THREAD

**Issue by adamabbate-q** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#4463]
Once a policy package is assigned to a user, it is possible to later remove this policy? Either through the Teams Admin Center or Powershell?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b3f54e11-f5b6-1ab9-6148-f053c66dd79b
* Version Independent ID: 27b514bf-1500-ef6d-ac11-e1860416ec04
* Content: [Manage policy packages in Microsoft Teams - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/manage-policy-packages#feedback)
* Content Source: [Teams/manage-policy-packages.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/manage-policy-packages.md)
* Service: **msteams**
* GitHub Login: @LanaChin
* Microsoft Alias: **v-lanac**

**Comment by scanum** [comment]
@adamabbate-q thank you for submitting feedback.
I think the best way forward is if you open a service ticket in your tenant so this can get resolved ASAP. Based on the outcome let me know if it is something that can be called out in the docs.
Please follow this link to contact support for business products:https://docs.microsoft.com/office365/admin/contact-support-for-business-products
Please keep us posted here on the resolution so we can feed whatever information you discover into the content.

**Comment by ChrisH82** [comment]
@adamabbate-q did you manage to get a resolution on this point or not - I have been playing with this and simalarly have the same issue with a policy package assigned to a user that i would prefer was reverted to none again

**Comment by adamabbate-q** [comment]
@ChrisH82 I have a ticket open with support at the moment, they originally provided some PowerShell cmdlets to fix this but that didn't work so they are still investigating. 

For the moment, I've just manually changed all the individual policies back to the default for the user I applied the Policy Package.

![image](https://user-images.githubusercontent.com/55467213/79510398-a758b800-8080-11ea-9c5e-a24a13dd673d.png)

The other suggestion from Support is to apply the "Small and medium business user (Business Voice)" policy instead but as I've done the above, I haven't gone down this path as yet.

![image](https://user-images.githubusercontent.com/55467213/79510519-d66f2980-8080-11ea-9c63-aac5a6cde001.png)

Will update the thread when I hear back from Support.
